src/core/public/mocks.ts

```diff
@@ -74,6 +74,7 @@ function createCoreSetupMock({
 } = {}) {
   const mock = {
     application: applicationServiceMock.createSetupContract(),
+    chrome: chromeServiceMock.createSetupContract(),
```

Comment:
So nice to have this in core mock, have encountered type error in some of the test cases